WordPress/src/main/java/org/wordpress/android/ui/main/WPMainNavigationView.kt

```diff
@@ -358,25 +358,24 @@ class WPMainNavigationView @JvmOverloads constructor(
 
     companion object {
         private val defaultPages = listOf(MY_SITE, READER, NEW_POST, ME, NOTIFS)
```

Comment:
maybe rename this to `oldPages`? or the `pages` to `newPages`

Reply:
Done here 39ad384822502a9720dded3f76cd37f057bb3ad7